plugins/push/frontend/public/javascripts/countly.models.js

```diff
@@ -192,6 +195,113 @@
                 periods: {daily: []},
             };
         },
+        getInitialBaseModel: function() {
```

Comment:
I have moved initial model state from view in helper object and ended up creating helper methods for each notificaiton type.